demisto_sdk/commands/generate_docs/generate_script_doc.py

```diff
@@ -74,8 +71,17 @@ def generate_script_doc(input, examples, output: str = None, permissions: str =
             doc.extend(generate_section('Permissions', ''))
 
         if used_in:
-            doc.extend(generate_list_section('Used In', used_in, True,
-                                             text='This script is used in the following playbooks and scripts.'))
+            if len(used_in) <= 10:
+                doc.extend(generate_list_section('Used In', used_in, True,
+                                                 text='This script is used in the following playbooks and scripts.'))
+            else:  # if we have more than 10 use a sample
+                print_warning(f'"Used In" section found too many scripts/playbooks ({len(used_in)}). Will use a sample of 10.'
+                              ' Full list is available as a comment in the README file.')
+                sample_used_in = random.sample(used_in, 10)
+                doc.extend(generate_list_section('Used In', sorted(sample_used_in), True,
+                                                 text='Sample usage of this script can be found in the following playbooks and scripts.'))
```

Comment:
why random?

Reply:
The idea is that when "Used In" is too big a list we limit to 10. I don't know which are the best 10 so just choose randomly.